content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -36,8 +36,8 @@ complete -o default -F __start_kubectl k
 ### ZSH
 
 ```bash
-source <(kubectl completion zsh)  # setup autocomplete in zsh into the current shell
-echo '[[ $commands[kubectl] ]] && source <(kubectl completion zsh)' >> ~/.zshrc # add autocomplete permanently to your zsh shell
+ # setup autocomplete in zsh into the current shell. add autocomplete permanently to your zsh shell
+source <(kubectl completion zsh) echo '[[ $commands[kubectl] ]] && source <(kubectl completion zsh)' >> ~/.zshrc 
```

Comment:
I think `source` and `echo` should be in different line, can the current version work?

Reply:
No echo and source should be on the same line. breaking them up results in snippets being constantly appended to the .rc file.